### PR TITLE
fix accessing parentStyleSheet from cssRules list

### PR DESCRIPTION
**Pull request by Ahmed-Hakeem**
Here myRules carry a list of type cssRulesList which has no parentStyleSheet inside of it, but instead we can access parentStyleSheet of any of its element.

Regarding  changing the log to be (myRules[0].parentStyleSheet == document.styleSheets[0]), because I think this will make the property value clear to the reader

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/web/api/cssrule/parentstylesheet/index.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ A {{domxref("StyleSheet")}} object.
 
 ```js
 let myRules = document.styleSheets[0].cssRules;
```
Comment (teoli2003) [inline files/en-us/web/api/cssrule/parentstylesheet/index.md:26]:
```suggestion
const docRules = document.styleSheets[0].cssRules;
```

File: files/en-us/web/api/cssrule/parentstylesheet/index.md [hunk 2]
```diff
@@ -24,7 +24,7 @@ A {{domxref("StyleSheet")}} object.
...
-console.log(myRules.parentStyleSheet);
+console.log(myRules[0].parentStyleSheet == document.styleSheets[0]); //returns true
```
Comment (teoli2003) [inline files/en-us/web/api/cssrule/parentstylesheet/index.md:27]:
```suggestion
console.log(docRules[0].parentStyleSheet == document.styleSheets[0]); // returns true 
```

**Review by teoli2003 — APPROVED** [review]
Two minor changes, and I'll merge this.

**Comment by teoli2003** [comment]
Thank you!

**Comment by Ahmed-Hakeem** [comment]
> Thank you!

My pleasure :)

**Pull request merged**